File: 3397-find-the-integer-added-to-array-i/find-the-integer-added-to-array-i.java
class Solution {
    public int addedInteger(int[] nums1, int[] nums2) {
        int sum=0;
        int sum1=0;
        for(int i=0;i<nums1.length;i++){
            sum+=nums1[i];
        }
        for(int i=0;i<nums2.length;i++){
            sum1+=nums2[i];
        }
        int r=sum1-sum;
        int n=nums1.length;
        return r/n;
    }
}

File: 3397-find-the-integer-added-to-array-i/find-the-integer-added-to-array-i.py
class Solution:
    def addedInteger(self, nums1: List[int], nums2: List[int]) -> int:
        sum2=0
        for i in range(len(nums1)):
            sum2+=nums1[i]
        sum1=0
        for j in range(len(nums2)):
            sum1+=nums2[j]
        d=sum1-sum2
        h=len(nums1)
        n=int(d/h)
        
        return n
